syncstorage/src/tokenserver/mod.rs

```diff
@@ -52,8 +50,8 @@ impl ServerState {
         let use_test_transactions = false;
 
         TokenserverPool::new(settings, &Metrics::from(&metrics), use_test_transactions)
-            .map(|db_pool| {
-                let service_id = db_pool
+            .map(|mut db_pool| {
+                db_pool.service_id = db_pool
```

Comment:
I suppose this could always potentially fail leaving `None` but I doubt it's even worth caring about. Maybe let's add a comment?

Reply:
Just making sure I understand, you mean a comment to explain the unlikelihood of the query failing?

Reply:
Yea, exactly

Reply:
What I meant was it could also happen due to say a temporary network hiccup

Reply:
Ah, got it